c/kastore.h

```diff
@@ -492,6 +495,9 @@ int kastore_puts_float64(
 
 /** @} */
 
+int kastore_bput(kastore_t *self, const char *key, size_t key_len, const void *array,
```

Comment:
What's the advantage of this vs using the existing ``kastore_put`` method? We can pass a flag ``KAS_BORROW_ARRAY`` to it, right?

Reply:
kastore_put calls kastore_oput which takes a non-const array, so even with a flag we would still need something like `bput`.

Reply:
That can be a static method though, not part of the public API, right?